### DISCUSSION TITLE
Create Geometry Tools Missing from Toolbox

### DISCUSSION THREAD

**Discussion by jgraebnerbond** [Dan-Patterson/npgeom#3] [General]
Hi Dan,

Thanks so much for creating this resource, it is really a massive help in working around the restrictive nature of the ESRI Licensing system.

I was hoping to use your split line at vertice tool but when I inspect the toolbox and then accompanying .py file I note that the Create Geometry toolset is missing?
![image](https://github.com/Dan-Patterson/numpy_geometry/assets/142955588/9849e8a8-3b53-48e0-ae75-ff1c82f9f10f)

Is there an alternate location I may access it from?

Thanks in advance!

**Comment by Dan-Patterson** [comment]
In the short term, it can be copies from and put in the npg folder with the rest
[npg_create location](https://github.com/Dan-Patterson/numpy_geometry/blob/master/arcpro_npg/npg/npg/npg_create.py)
